ffi/diplomat/js/examples/node/build.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+MAKEFILE_NIGHTLY="${MAKEFILE_NIGHTLY:-nightly-2022-04-05}"
```

Comment:
Question:

1. Why the extra `-` before "nightly"?
2. Does this work in all shells or only Bash?

Reply:
`-` is how you give it a default value. it seems to work in `sh` at least